codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/customization/DisabledAuth.kt

```diff
@@ -15,14 +15,39 @@ internal val DISABLED_AUTH_OPERATIONS: Map<String, Set<String>> = mapOf(
         "com.amazonaws.sts#AssumeRoleWithSAML",
         "com.amazonaws.sts#AssumeRoleWithWebIdentity"
     ),
+    "com.amazonaws.cognitoidentity#AWSCognitoIdentityService" to setOf(
```

Comment:
worth adding link in comments to https://docs.aws.amazon.com/cognito/latest/developerguide/security_iam_service-with-iam.html for context